src/components/nav/Navigation.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import style from './Navigation.module.scss'
+
+export function Navigation() {
```

Comment:
your component in most of the cases must be data driven. in this case you need to extract links into an array and then iterate over it to produce the markup

```tsx
const links = [ { title: 'Blog', href: '#' } ]

retrun (<ul>{links.map((link) => <li><a href={link.href}>{link.title}</li>)}</ul>)
```